exercises/edward_brown/ex01_basics/include/MakeWordFrequencyMap.h

```diff
@@ -0,0 +1,46 @@
+#ifndef EX1_MAKE_WORD_FREQUENCY_MAP
+#define EX1_MAKE_WORD_FREQUENCY_MAP
+#include "ForEachWord.h"
+#include "Punctuation.h"
+#include "ToLowerCase.h"
+#include <unordered_map>
+
+/** A predicate which determines if CandidateWord is a word for the purposes of
+ *  word frequency analysis.
+ *
+ *  @param CandidateWord The candidate to check.
+ *  @return True if CandidateWord is a 'word' false if not.
+ */
+bool isWord(std::string const &CandidateWord) {
```

Comment:
Pleased to hear it, it may be something worth adding to the wiki pages. I thought the [LLVM](http://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly) convention was a little odd but fell back on it as that's what the standards say to do.

> Variable names should be nouns (as they represent state). The name should be camel case, and start with an upper case letter (e.g. Leader or Boats).